input/pagecontent/vaccination.md

```diff
@@ -47,7 +47,7 @@ Implementers SHALL use at least one of the following code systems for identifyin
 Some additional information about the table above:
 
 * A **code system** defines a set of "codes" and their mappings onto specific meanings. For example, the [CVX code system][CVX] includes the code `207`, which is mapped to the meaning "COVID-19, mRNA, LNP-S, PF, 100 mcg/0.5 mL dose". Using a code system helps to ensure implementers represent a given concept in a consistent way that other implementers can understand.
-* A **value set** defines a group of codes that can be used within a specific element. For example, we define a [value set of SNOMED-CT codes for vaccine products][VaccineTypeSNOMED] for use in the `vaccineCode` element. FHIR allows an element to be "bound" to a value set, meaning that the contents of that element must come from the value set (for a "[required binding][binding]") or should come form the value set (for a "[preferred binding][binding]").
+* A **value set** defines a group of codes that can be used within a specific element. For example, we define a [value set of SNOMED-CT codes for vaccine products](VaccineTypeSNOMED) for use in the `vaccineCode` element. FHIR allows an element to be "bound" to a value set, meaning that the contents of that element must come from the value set (for a "[required binding][binding]") or should come form the value set (for a "[preferred binding][binding]").
```

Comment:
(VaccineTypeSNOMED) does not work.

Reply:
See https://raw.githubusercontent.com/HL7/fhir-shc-vaccination-ig/dev/input/includes/markdown-link-references.md  perhaps [VaccineSNOMED] would work